tests/core/fs/old_storage/manifest_storage.py

```diff
@@ -113,7 +112,7 @@ async def _create_db(self) -> None:
                 );
                 """
             )
-            # Set the default "prevent sync" pattern if it doesn't exist
+            # set the default "prevent sync" pattern if it doesn't exist
```

Comment:
Typing renaming `Set -> set` has made some collateral damages ^^

(also see above for other comments & docstring with same change)